src/components/synchronization-page/OrganisationUnitPage.jsx

```diff
@@ -15,7 +15,11 @@ export default class OrganisationUnitPage extends React.Component {
         const title = i18n.t("Organisation Units Synchronization");
 
         return (
-            <GenericSynchronizationPage d2={d2} models={[OrganisationUnitModel]} title={title} />
+            <GenericSynchronizationPage
+                d2={d2}
+                models={[OrganisationUnitModel, OrganisationUnitGroupModel]}
```

Comment:
Following the rule of thumb of not using created-on-the-fly values for props, we can use use a `this.models` like we do in `OrganisationUnitPage`.